.eslintrc

```diff
@@ -1,13 +1,20 @@
 {
   "env": {
     "browser": true,
-    "es6": true
+    "es6": true,
+    "node": true
   },
   "extends": [
     "eslint:recommended",
     "google"
   ],
   "rules": {
-    "no-var": 0,
+    "no-var": 0
```

Comment:
Is this needed? Since you're only testing in puppeteer, I think it's safe to use `let`/`const` everywhere, right?

Reply:
Yes, but `no-var` was there earlier. I removed the ending comma(,).